### PR TITLE
hunterize 2020.1

### PR DESCRIPTION
**Pull request by elisemorysc**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -24,11 +24,26 @@ if (POLICY CMP0054)
...
+set(SPIRV-Headers_SOURCE_DIR ${SPIRV-HEADERS_ROOT})
+
 include(GNUInstallDirs)
+if(NOT HUNTER_ENABLED)
+  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+  set(CMAKE_CXX_STANDARD 11)
+endif(NOT HUNTER_ENABLED)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_CXX_STANDARD 11)
```
Comment (rbsheth) [inline CMakeLists.txt]: Remove?

File: CMakeLists.txt [hunk 2]
```diff
@@ -330,10 +345,39 @@ add_custom_target(spirv-tools-shared-pkg-config ALL
...
+  include(CMakePackageConfigHelpers)
+  set(SPIRV_CONFIG_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/SPIRV-Tools)
+
   install(
     FILES
       ${CMAKE_CURRENT_BINARY_DIR}/SPIRV-Tools.pc
       ${CMAKE_CURRENT_BINARY_DIR}/SPIRV-Tools-shared.pc
     DESTINATION
       ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+
```
Comment (rbsheth) [inline CMakeLists.txt:355]: Remove extra line

File: source/val/validate_conversion.cpp [hunk 3]
```diff
@@ -469,7 +469,7 @@ spv_result_t ConversionPass(ValidationState_t& _, const Instruction* inst) {
                << "or scalar: " << spvOpcodeString(opcode);
 
       if (_.version() >= SPV_SPIRV_VERSION_WORD(1, 5) ||
-          _.HasExtension(kSPV_KHR_physical_storage_buffer)) {
+          _.HasExtension(kSPV_EXT_physical_storage_buffer)) {
```
Comment (rbsheth) [inline source/val/validate_conversion.cpp:472]: Is this needed?

File: CMakeLists.txt (hunk 1 above)
Comment (elisemorysc) [reply CMakeLists.txt]: Yes

File: source/val/validate_conversion.cpp (hunk 3 above)
Comment (elisemorysc) [reply source/val/validate_conversion.cpp:472]: Yes, it was modified in the headers somehow. Couldn't pin the exact place it was changed

**Review by rahulshethsc — APPROVED** [review]

**Pull request merged**